### PR TITLE
HPCC-16162 Parse out ESDLService name for UI consumption

### PR DESCRIPTION
**Pull request by rpastrana**
- Parses out, and reports ESDSLService in getESDLDefinition response
- Parses out, and reports ESDSLService in listDESDLEspBindings response

Signed-off-by: rpastrana rodrigo.pastrana@lexisnexis.com

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx please review.

File: esp/scm/ws_esdlconfig.ecm [hunk 1]
```diff
@@ -41,6 +41,7 @@ ESPrequest [nil_remove] GetESDLDefinitionRequest
 ESPresponse [exceptions_inline] GetESDLDefinitionResponse
 {
     string Id;
+    ESParray<string, Name> ESDLServices;
```
Comment (wangkx) [inline esp/scm/ws_esdlconfig.ecm]: Do you need a version here? The same as line 110.

**Comment by wangkx** [comment]
@rpastrana looks good. Just one question.

**Comment by rpastrana** [comment]
@wangkx please review.

**Comment by wangkx** [comment]
@rpastrana looks fine.

**Pull request merged**